docs/testplan/loss_threshold_test.md

```diff
@@ -0,0 +1,45 @@
+# SONiC Switch Loss Threshold Test
+
+- [SONiC Switch Loss Threshold Test](#sonic-switch-loss-threshold-test)
+  - [Test Objective](#test-objective)
+  - [Test Setup](#test-setup)
+  - [Test Steps](#test-steps)
+  - [Metrics Processing](#metrics-processing)
+
+## Test Objective
+
+This test aims to determine the maximum traffic rate that results in 0% packet loss across different packet sizes. By identifying this threshold, we can assess the switch's forwarding capability and validate its performance under various traffic conditions.
+
+## Test Setup
+
+This test is designed to be flexible and applicable to various network setups for evaluating traffic loss thresholds. It can be used to assess different configurations, such as one-tier and multi-tier networks. If testing the setup described in `BGP_IPv6_test.md`, ensure the testbed is properly configured by following the procedures outlined in that plan.
+
+1. **Establish BGP Sessions and Advertise Routes**
+   Follow the steps in the multi-tier BGP test plan to:
+   - Configure the required IPv6 BGP sessions between the DUT and its neighbors.
+   - Advertise the intended set of IPv6 routes to ensure full route visibility.
+
+2. **Define Full-Mesh Traffic Flows**
+   - Set up full-mesh traffic flows on the traffic generators so that all DUT ports are actively involved.
+   - Target all advertised routes to maximize coverage.
```

Comment:
is this a real requirement?  this is not route scale test.

In this case, all we need to make sure is each port advertises it own route.